conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleInterpreterGroupExplode.scala

```diff
@@ -118,23 +116,36 @@ case class MappingRuleInterpreterGroupExplode(rule: MappingConformanceRule,
     errorsDf
   }
 
-  private def validateMappingRule(df: Dataset[Row],
-                                  dao: EnceladusDAO,
-                                  mappingTableDef: MappingTable,
-                                  mapTable: Dataset[Row],
-                                  joinContidionStr: String,
-                                  defaultMappingValueMap: Map[String, String])
-                                 (implicit spark: SparkSession): Unit = {
-    // validate the default value against the mapping table schema
-    if (defaultMappingValueMap.contains(rule.targetAttribute)) {
+  private def getDefaultValue(mappingTableDef: MappingTable)
```

Comment:
This seems to be a complete duplicate of the method in `MappingRuleInterpreter`, can we abstract it out to reduce duplication?

Reply:
Ok, will do. the idea was that one of the mapping rule classes will be removed at some point, just didn't want for these 2 implementations to have common dependencies.